docs/examples/hosting/docker/caddy/Caddyfile

```diff
@@ -0,0 +1,28 @@
+localhost {
+    encode gzip zstd
+
+    handle_path /server* {
```

Comment:
`handle_path` instead of `handle` so the `/server` path isn't sent with the request to the proxy.